services/m3dbnode/config/config_test.go

```diff
@@ -225,19 +226,35 @@ pooling:
               highWatermark: 0.02
 
 config:
-  service:
-      env: production
-      zone: us-west1
-      service: m3dbnode
-      cacheDir: /var/lib/m3kv
-      etcdClusters:
-          - zone: us-west1
-            endpoints:
-                - etcd01-us-west1:2379
-                - etcd02-us-west1:2379
-                - etcd03-us-west1:2379
-                - etcd04-us-west1:2379
-                - etcd05-us-west1:2379
+    service:
+        env: production
+        zone: embedded
+        service: m3db
+        cacheDir: /var/lib/m3kv
+        etcdClusters:
+            - zone: embedded
+              endpoints:
+                  - 1.1.1.1:2379
+                  - 1.1.1.2:2379
+                  - 1.1.1.3:2379
+    seedNode:
+        listenPeerUrls:
+            - http://0.0.0.0:2380
+        listenClientUrls:
+            - http://0.0.0.0:2379
+        rootDir: /var/lib/etcd
+        initialAdvertisePeerUrls:
+            - http://1.1.1.1:2380
+        advertiseClientUrls:
+            - http://1.1.1.1:2379
+        initialCluster:
+            - hostId: host1
+              endpoint: http://1.1.1.1:2380
+            - hostId: host2
+              endpoint: http://1.1.1.2:2380
+            - hostId: host3
+              endpoint: http://1.1.1.3:2380
+        name: host1
```

Comment:
Hm this is meant to be the local `HostID`? Of this node?  If so we'll need to pass this in, as the top level config already has this and we won't want user's to have to define the local host ID twice:
https://github.com/m3db/m3db/blob/master/services/m3dbnode/config/config.go#L57

Reply:
Yep, makes sense. Etcd config requires that the "seed node" `name` must equal one of the `name`s specified in the initialCluster config. After splitting the initialCluster into hostID/endpoint values, it makes more sense to just use the HostID of the node.